components/css/componentPage.scss

```diff
@@ -15,6 +15,15 @@
   margin-bottom: 73px;
 }
 
+:local(.compStatus) {
+  min-height: 0 !important;
```

Comment:
Can we do this without `!important`?

Reply:
If we redefine it for `.spectrum--large .compStatus` (which overrides it otherwise), but TBH, this is a hack, and `!important` makes that very clear :)

Reply:
@GarthDB explored fixing this using CSS variable overrides, but we can't do it as the component references global variables and does not differentiate between left and right padding on the pseudo element. It's either get more specific or be `!important`, and TBH i think it's fine to be `!important` as this is a hack for this specific use case.

Reply:
@GarthDB updated to explicitly define `.spectrum--large` etc and removed the `!important`s.